Sources/HomomorphicEncryption/Context.swift

```diff
@@ -128,19 +129,19 @@ public final class Context<Scheme: HeScheme>: Equatable, Sendable {
     ///   - rhs: Another context to compare.
     /// - Returns: Whether or not the two contexts are equal.
     @inlinable
-    public static func == (lhs: Context<Scheme>, rhs: Context<Scheme>) -> Bool {
+    public static func == (lhs: Context, rhs: Context) -> Bool {
         lhs === rhs || lhs.encryptionParameters == rhs.encryptionParameters
     }
 
     @inlinable
-    func getRnsTool(moduliCount: Int) -> RnsTool<Scalar> {
+    public func _getRnsTool(moduliCount: Int) -> _RnsTool<Scalar> {
         precondition(moduliCount <= rnsTools.count && moduliCount > 0, "Invalid number of moduli")
         return rnsTools[rnsTools.count - moduliCount]
     }
 }
 
 extension Context: CustomStringConvertible {
     public var description: String {
-        "Context<\(Scheme.self)>(encryptionParameters=\(encryptionParameters.description))"
+        "Context<\(Scalar.self)>(encryptionParameters=\(encryptionParameters.description))"
```

Comment:
Revert to using scheme.
```suggestion
        "Context<\(Scheme.self)>(encryptionParameters=\(encryptionParameters.description))"
```